### ISSUE TITLE
First Time Build Errors

### ISSUE THREAD

**Issue by Bradleycorn** [Sitata/titanium-google-analytics#19]
I ran into a couple issues when trying to compile the android module source for the first time. Might be helpful to others to add them the the readme. I'm adding them here in case others are having similar trouble.

1.) You have to create your own build.properties file. There is an example file in the repo (build.properties.example), but you have to copy it to a new file named build.properties and edit it's values to point to the various titanium and android sdk's on your system. As a novice when it comes to titanium modules, this was not readily apparent to me. 

2.) After setting up my build properties, the build failed with an error message that the <path to project>/build/docs folder doesn't exist. I checked and sure enough it did not. So I created the folder, and ran the build again, and it all worked fine, hooray! Again I'm a module novice, so I'm not sure if just creating the docs folder was the "right" thing to do, but it worked, so I'm happy.

**Comment by astjohn** [comment]
@Bradleycorn, Thanks for pointing that stuff out and I'm glad you managed to get it to work. In the future, you actually don't have to compile the module yourself at all. You can simply download the zip file and unzip it in your project's root folder. The zip files for most modules are usually located at `/dist/` for android and in the root of the modules folder for ios.

In this project, I have also attached the release zip files to the project's [releases list](https://github.com/Sitata/titanium-google-analytics/releases)

That said, now that you're compiling your own modules and such, I'd recommend trying to create one on your own. It's not too bad and it's always nice to have others help out.

Cheers!

**Comment by Bradleycorn** [comment]
Thanks @astjohn. I actually wanted to build from source as I forked the project in case I want to add some of my own customizations. Prior to your picking this project up from the previous author, I had taken his initial work and basically rewrote it to use updated api's from google, so I'm somewhat familiar with both this project, and creating modules. I tried awhile back to write a module for Chromecasting, but at the time there were all sorts of issues relating to including jars with resources and such. It's been awhile now, so perhaps those issues are resolved and I'll pick that up again.
